paper.md

```diff
@@ -0,0 +1,84 @@
+tags:
+  - Python
+  - quantum
+  - quantum computer
+  - pulse
+  - control
+authors:
+  - name: Daniel Puzzuoli
+    corresponding: true
+    affiliation: 1
+  - name: Christopher J. Wood
+    affiliation: 2
+  - name: Daniel J. Egger
+    affiliation: 3
+  - name: Benjamin Rosand
+    affiliation: 4
+  - name: Kento Ueda
+    affiliation: 5
+  - name: Ian Hincks
+    affiliation: 6
+  - name: Haggai Landa
+    affiliation: 7
+  - name: Moein Malekakhlagh
+    affiliation: 2
+  - name: Avery Parr
+    affiliation: 8
+  - name: R. K. Rupesh
+    affiliation: 9
+  - name: Matthew Treinish
+    affiliation: 2
+affiliations:
+ - name: IBM Quantum, IBM Canada, Vancouver, BC, Canada
+   index: 1
+ - name: IBM Quantum, IBM T.J. Watson Research Center, Yorktown Heights, NY, USA
+   index: 2
+ - name: IBM Quantum, IBM Research Europe - Zurich, Ruschlikon, Switzerland
+   index: 3
+ - name: Department of Physics, Yale University, New Haven, CT, USA
+   index: 4
+ - name: IBM Quantum, IBM Research Tokyo, Tokyo, Japan
+   index: 5
+ - name: IBM Quantum, IBM Canada, Markham, ON, Canada
+   index: 6
+ - name: IBM Quantum, IBM Research Israel, Haifa, Israel
+   index: 7
+ - name: Department of Physics, Harvard University, Cambridge, MA, USA
+   index: 8
+ - name: School of Physics, University of Hyderabad, Hyderabad, India
+   index: 9
+
+
+date: 31 July 2023
+bibliography: paper.bib
+---
+
+# Summary
+
+Qiskit Dynamics is an open source Python library for numerically simulating the time dynamics of finite-dimensional quantum systems. The goal of the package is to provide flexible configuration of the numerical methods used for simulation: general tools for transforming models of quantum systems for more efficient simulation (rotating frames and the rotating wave approximation), choice of array representations (dense v.s. sparse, and different array libraries), and access to different types of underlying solvers (standard ODE v.s. geometric solvers). The package also contains advanced functionality for computing time-dependent perturbation theory expressions used in robust quantum control optimization `[@perturb1; @perturb2]`. 
+
+As part of the Qiskit Ecosystem (https://qiskit.org/ecosystem), the package interfaces with other parts of Qiskit `[@Qiskit]`. Most notably, Qiskit Dynamics provides tools for simulating control sequences specified by Qiskit Pulse `[@alexander_qiskit_2020]`, which is used to specify hardware-level control of quantum computers. Higher level interfaces allow users to build and interact with simulation-based objects that target the same constraints (coupling map, timing, etc.) as a specified IBM Quantum computer.
+
+Lastly, to facilitate high-perfomance applications, Qiskit Dynamics is compatible with the JAX array library `[@jax2018github]`. As such, all core computations are just-in-time compilable, automatically differentiable, and executable on GPU.
+
+# Statement of need
```

Comment:
If I were a ref I would ask: Don't we already have all this in Qutip? Why do we need Qiskit Dyamics?

I think you could make a case around the Qiskit Pulse compatibility and saying that Qiskit Dynamics also naturally integrates in the Qiskit ecosystem. Or sth along those lines.

Reply:
After reading the journal requirements and examples, I don't actually think there is any need to justify why a new package was written v.s. using something pre-existing. The guidelines for authors, as far as I can tell, don't say anything about explaining your package within the context of other work (aside from citing it). I don't mean this as a denial that this is a generally important point; just explaining why I wrote this section in this way.

So, with this in mind, I wrote the "statement of need" section as a simple description of "why do we need these kinds of simulations/interfaces", rather than a "why do we need these kinds of simulations/interfaces AND why did we write another package". It may be high level, but everything I say in that paragraph does implicitly explain why Dynamics was written v.s. just using qutip. I do also describe qiskit integration in the second paragraph of this section. 

What are your thoughts on this? We could rearrange it to highlight the qiskit integration first, as it is the most obviously unique feature, then describe the other technical stuff after.